### DISCUSSION TITLE
How can I get the creator of smart contract ?

### DISCUSSION THREAD

**Discussion by hientrangg** [0xPolygon/polygon-edge#703] [Q&A]
Hi, 
I want to use Edge to make a chain, during the process, I found a problem. Can you teach me how (where) that I can get the creator of a  deployed smart contract ?

**Comment by laviniat1996** [comment]
@hientrangg Thank you for your question!

If you have the transaction hash, you can check the creator of a smart contract by querying the receipt of the transaction that did the deployment, and fetching the `result.From` field.
Another way would be by using a block explorer, or by indexing the transactions on the blockchain itself.

**Reply by hientrangg** [reply]
Thank you, have a nice day.
